frontend/src/metabase/query_builder/components/expressions/CombineColumns/ColumnAndSeparatorRow/ColumnInput.module.css

```diff
@@ -0,0 +1,20 @@
+.button {
+  justify-content: space-between !important;
+  font-weight: normal;
+}
+
+button.root {
```

Comment:
`button` is here for specificity, to avoid `!important` in the properties that are being overwritten.

Reply:
here is a [thread](https://metaboat.slack.com/archives/C0645JP1W81/p1712670698147929)

Reply:
Do you want me to set up a custom variant for this? 

This variant is only going the be used in this instance, so I think might be overkill.